### ISSUE TITLE
A way to issue warnings

### ISSUE THREAD

**Issue by jamestalmage** [SamVerschueren/listr#16]
Swallowing all output may encourage users to ignore warnings which maybe they shouldn't (linter warnings, etc).

It would be nice if tasks were able to push any number of warnings to the output, that were not swallowed by `log-update`

**Comment by SamVerschueren** [comment]
So you mean like the observable thing we have now, but instead of overwriting that output, it just adds a new line?

**Comment by jamestalmage** [comment]
Yes. Not sure how that would work with subtasks. Maybe they don't collapse if a warning is made.

**Comment by okonet** [comment]
I ran into this with the https://github.com/okonet/lint-staged/pull/32 where I'm using `spawn` to log linting errors. The issue is that some linter's output is still swallowed and the Listr output displayed twice.

Is there a way to still output to stdout from my tasks? Right now it look like this:

<img width="624" alt="2016-07-22 at 23 24" src="https://cloud.githubusercontent.com/assets/11071/17071700/8508becc-5063-11e6-9a65-d10aaf35ddf3.png">

**Comment by ben-eb** [comment]
+1 from me for not swallowing error messages.

https://gitter.im/avajs/ava?at=57accc4b613cc180367b0ab5

**Comment by okonet** [comment]
Any update on this one? I'd like to make a decision regarding this library in https://github.com/okonet/lint-staged/pull/32 and ATM this issue is what prevents me from merging it.

**Comment by SamVerschueren** [comment]
@okonet Sorry for the delay on this issue. The render refactoring is currently in development and hope to ship it soon, although I also have other things on my mind lately. Can't give you an exact timeframe though...

**Comment by okonet** [comment]
@SamVerschueren do you think #23 can also be fixed during this refacroting?

**Comment by SamVerschueren** [comment]
@okonet Yes sure, see my comment in the thread.

**Comment by SamVerschueren** [comment]
We now have custom renderers, so it would be possible to write a renderer that does this. The current default ([listr-update-renderer](https://github.com/SamVerschueren/listr-update-renderer)) just outputs like it did previously.
